src/ensembl/src/content/app/browser/Browser.test.tsx

```diff
@@ -70,43 +76,45 @@ describe('<Browser />', () => {
     jest.resetAllMocks();
   });
 
-  const wrappingComponent = (props: any) => (
-    <MemoryRouter>{props.children}</MemoryRouter>
-  );
-
   const mountBrowserComponent = (props?: Partial<BrowserProps>) =>
-    mount(<Browser {...defaultProps} {...props} />, { wrappingComponent });
+    render(
+      <MemoryRouter>
+        <Browser {...defaultProps} {...props} />
+      </MemoryRouter>
+    );
 
   describe('rendering', () => {
     test('does not render when no activeGenomeId', () => {
-      const wrapper = mountBrowserComponent({ activeGenomeId: null });
-      expect(wrapper.html()).toBe(null);
+      const { container } = mountBrowserComponent({ activeGenomeId: null });
+      expect(container.innerHTML).toBeFalsy();
     });
 
     test('renders links to example objects only if there is no selected focus feature', () => {
-      const wrapper = mountBrowserComponent();
-      expect(wrapper.find(ExampleObjectLinks)).toHaveLength(1);
+      let { container } = mountBrowserComponent();
+
+      expect(container.querySelectorAll('.exampleLinks')).toHaveLength(1);
 
-      wrapper.setProps({
+      container = mountBrowserComponent({
         browserQueryParams: {
           focus: faker.lorem.words()
         }
-      });
-      expect(wrapper.find(ExampleObjectLinks)).toHaveLength(0);
+      }).container;
+
+      expect(container.querySelectorAll('.exampleLinks')).toHaveLength(0);
     });
 
     test('renders the genome browser and track panel only when there is a selected focus feature', () => {
-      const wrapper = mountBrowserComponent();
+      let { container } = mountBrowserComponent();
 
-      expect(wrapper.find(BrowserImage)).toHaveLength(0);
-      expect(wrapper.find(TrackPanel)).toHaveLength(0);
+      expect(container.querySelectorAll('.browserImage')).toHaveLength(0);
+      expect(container.querySelectorAll('.trackPanel')).toHaveLength(0);
 
-      wrapper.setProps({
+      container = mountBrowserComponent({
```

Comment:
The direct equivalent to enzyme's `setProps` is the `rerender` method on the object returned from the `render` function. Don't know if it's better any better here than redefining the container; but just thought I'd mention this possibility.

Reply:
Updated it to use `rerender`